### PR TITLE
fix: Replace identify method strings with constants

### PR DESCRIPTION
**Pull request by alexs-mparticle**
## Instructions
 1. PR target branch should be against `development`
 2. PR title name should follow this format: https://github.com/mParticle/mparticle-workflows/blob/main/.github/workflows/pr-title-check.yml
 3. PR branch prefix should follow this format: https://github.com/mParticle/mparticle-workflows/blob/main/.github/workflows/pr-branch-check-name.yml

 ## Summary
 - Replace Identify method strings with constants
 - Also removes an unused parameter from `setForwarderOnUserIdentified`

 ## Testing Plan
 - [x] Was this tested locally? If not, explain why.
 - Using a sample app, verify that Identify, Modify, Login and Logout have no changes in behavior.

 ## Reference Issue (For mParticle employees only.  Ignore if you are an outside contributor)
 - Closes https://go.mparticle.com/work/SQDSDKS-6029

### CODE REVIEW

File: src/forwarders.js [hunk 1]
```diff
@@ -440,6 +443,7 @@ export default function Forwarders(mpInstance, kitBlocker) {
         });
     };
 
+    // TODO: This is a v1 method, we should deprecate
```
Comment (rmi22186) [inline src/forwarders.js:446]: Please instead make this a ticket and add the ticket instead of this description.

File: src/identity.js [hunk 2]
```diff
@@ -1620,6 +1622,8 @@ export default function Identity(mpInstance) {
                             mpInstance._APIClient.prepareForwardingStats
                         );
                     }
+
+                    // TODO: This is a v1 method. We should deprecate
```
Comment (rmi22186) [inline src/identity.js:1626]: Please instead make this a ticket and add the ticket instead of this description

**Review by rmi22186 — CHANGES_REQUESTED** [review]
Small comments otherwise looks good.

File: src/identity.js (hunk 2 above)
Comment (alexs-mparticle) [inline src/identity.js:1626]:
```suggestion
                    // TODO: https://go.mparticle.com/work/SQDSDKS-6036
```

File: src/forwarders.js (hunk 1 above)
Comment (alexs-mparticle) [inline src/forwarders.js:446]:
```suggestion
    // TODO: https://go.mparticle.com/work/SQDSDKS-6036
```

**Review by rmi22186 — APPROVED** [review]

**Pull request merged**